swift/ql/src/queries/Security/CWE-259/ConstantPassword.qhelp

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+  <overview>
+    <p>Deriving password-based encryption keys using hard-coded passwords is insecure, because the generated key may be easily discovered. Data hashed using constant salts are vulnerable to dictionary attacks, enabling attackers to recover the original input.</p>
```

Comment:
I think the word data is generally treated as singular, but ok for you to leave as-is if you have strong feelings about this 😄 
 
```suggestion
    <p>Deriving password-based encryption keys using hard-coded passwords is insecure, because the generated key may be easily discovered. Data hashed using constant salts is vulnerable to dictionary attacks, enabling attackers to recover the original input.</p>
```

Reply:
I don't :D Changed as suggested.